packages/hooks/src/useAuthorize.ts

```diff
@@ -1,14 +1,15 @@
-import { useMemo } from 'react';
+import { getActiveToken } from '@deriv/utils';
 import { useFetch } from '@deriv/api';
-import { useStore } from '@deriv/stores';
+import { useMemo } from 'react';
 
 /** A custom hook that authorize the user with the given token. If no token is given, it will use the current token. */
 const useAuthorize = (token?: string) => {
-    const { client } = useStore();
-    const { accounts, loginid = '' } = client;
-    const current_token = accounts[loginid || '']?.token;
+    const current_token = getActiveToken();
 
-    const { data, ...rest } = useFetch('authorize', { payload: { authorize: token || current_token } });
+    const { data, ...rest } = useFetch('authorize', {
+        payload: { authorize: token || current_token },
+        options: { refetchOnMount: false, keepPreviousData: true },
```

Comment:
@farzin-deriv Ok, I'll delete `refetchOnMount`, but we need `keepPreviousData` to prevent loading when the user switches between wallets 1st time (for desktop and moblie)